### ISSUE TITLE
Nil bug in version detection

### ISSUE THREAD

**Issue by alethea** [fnando/browser#229]
## Description

Attempting to use platform version checking for Android fails on at least one version of Firefox for Android.

## How to reproduce

```ruby
browser = Browser.new("Mozilla/5.0 (Android; Mobile; rv:40.0) Gecko/40.0 Firefox/40.0")
browser.platform.android?(5)
```

## What do you expect

I expected this to fail silently. Comparing 5 to nil should be false.

## What happened instead

An argument error was raise from RubyGems for attempting to compare the requirement against a nil version.

## Software:

- Rails version: 4.2.5.1
- Browser gem version: 2.0.2

## Full backtrace

```text
ArgumentError: Need a Gem::Version: nil
/usr/lib64/ruby/site_ruby/2.2.0/rubygems/requirement.rb:239 :in `satisfied_by?`
[GEM_ROOT]/gems/browser-2.0.2/lib/browser/detect_version.rb:13 :in `detect_version?`
[GEM_ROOT]/gems/browser-2.0.2/lib/browser/platform.rb:38 :in `android?` 
```

## Report
http://user-agent.herokuapp.com/?ua=Mozilla%2F5.0+%28Android%3B+Mobile%3B+rv%3A40.0%29+Gecko%2F40.0+Firefox%2F40.0&accept_language=en-US%2Cen%3Bq%3D0.5

**Comment by fnando** [comment]
Hi @alethea! Can you see if #230 works as expected for you?

**Comment by alethea** [comment]
Works for me. Thanks for the quick turnaround!

**Comment by fnando** [comment]
Just released v2.0.3. ;)

**Comment by alethea** [comment]
Awesome, thanks so much!
